docs/ref/databases.txt

```diff
@@ -367,28 +367,6 @@ this for you (the database backend layer and the model population layer are
 separated internally so the database layer doesn't know it needs to make this
 conversion in this one particular case).
 
-If you're using MySQLdb 1.2.1p2, Django's standard
-:class:`~django.db.models.CharField` class will return unicode strings even
-with ``utf8_bin`` collation. However, :class:`~django.db.models.TextField`
-fields will be returned as an ``array.array`` instance (from Python's standard
-``array`` module). There isn't a lot Django can do about that, since, again,
-the information needed to make the necessary conversions isn't available when
-the data is read in from the database. This problem was `fixed in MySQLdb
-1.2.2`_, so if you want to use :class:`~django.db.models.TextField` with
-``utf8_bin`` collation, upgrading to version 1.2.2 and then dealing with the
-bytestrings (which shouldn't be too difficult) as described above is the
-recommended solution.
-
-Should you decide to use ``utf8_bin`` collation for some of your tables with
```

Comment:
After investigation, it seems obsolete based on the discussion about the utf8_bin paragraph being obsolete, but let us know if you have a better understanding of this issue!